core/simulations/configs/vaultLiquidation.ts

```diff
@@ -19,7 +22,9 @@ const simulation: Simulation = {
         {
             title: 'Reset blockchain fork',
             entry: async () => {
-                await resetNetworkAndSetupWallet();
+                const latestBlock = await hre.ethers.provider.getBlockNumber();
```

Comment:
Can you please move it to the `promptToGetBlockNumber` 😉 ?
So that you only fetch it if `max` is undefined. I would also: 
- rename `promptToGetNumber` to `promptToGetBlockNumber`
- move `title` to there and make it a default
- restructure arguments to be a single `options` objects, since most of them are optional